src/overlays/actors/ovl_En_Dodojr/z_en_dodojr.h

```diff
@@ -16,11 +16,11 @@ typedef struct EnDodojr {
     /* 0x01E0 */ Actor* bomb;
     /* 0x01E4 */ Vec3f headPos;
     /* 0x01F0 */ Vec3f dustPos;
-    /* 0x01FC */ s16 unk_1FC;
-    /* 0x01FE */ s16 timer1;
-    /* 0x0200 */ s16 timer2;
-    /* 0x0202 */ s16 timer3;
-    /* 0x0204 */ s16 timer4;
+    /* 0x01FC */ s16 bounceCounter; // Also used for flashing when dying.
```

Comment:
unfortunately I think this should be a more generic name to fit the "flashing" usage, maybe "counter" :/

Reply:
This is kind of a broader problem I am still a little unclear on, when something feels re-used for a different purpose out of convenience - should the name reflect what it was likely intended for, or try to capture all uses? Making it generic you sometimes lose information that would be useful to have. In this case I think you're right it should just be `counter`, but I've been struggling with this lately.

[eb4e12c](https://github.com/zeldaret/oot/pull/1311/commits/eb4e12c161cef3feada227ecf0260b48faa7a375)